assets/queries/dockerfile/missing_dnf_clean_all/query.rego

```diff
@@ -18,6 +18,30 @@ CxPolicy[result] {
 	}
 }
 
+CxPolicy[resource.StartLine] {
+	resource := input.document[i].command[name][_]
+	resource.Cmd == "run"
+	command := resource.Value[0]
+
+	containsInstallCommand(command)
+	containsDnfClean(input.document[i].command[name], resource.StartLine) == false
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("FROM={{%s}}.RUN={{%s}}", [name, resource.Value[0]]),
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": "After installing a package with dnf, command 'dnf clean all' is run.",
+		"keyActualValue": "Command `dnf clean all` should be run after installing packages.",
```

Comment:
ActualValue should reflect what is happening with the resource. In this case, the clean command is not being run. Actually, this comment is meant for the check above, in line 17, since it seems to me this one is redundant, but GitHub didn't allow the comment there.